features/step_definitions/files_steps.rb

```diff
@@ -26,7 +41,6 @@
 end
 
 
-Then(/^I (?:still|again) have an uncommitted file with name: "([^"]+)" and content: "([^"]+)"$/) do |file_name, content|
-  expect(uncommitted_files).to eql [file_name]
-  expect(IO.read file_name).to eql content
+Then(/^I (?:still|again) have an uncommitted file with name: "([^"]+)" and content: "([^"]+)"$/) do |name, content|
```

Comment:
Sure, no problem. Don't worry, I'm using find+sed which is maybe a few keystrokes slower :) . Are you using a plugin to do something like this in Vim?